packages/jest-circus-allure-reporter/README.md

```diff
@@ -0,0 +1,152 @@
+# Jest Circus Allure Reporter
+
+[![jest](https://jestjs.io/img/jest-badge.svg)](https://github.com/facebook/jest)
+
+A Jest Circus compatible reporter for Allure.
+
+Most of the code is forked from  [ryparker/jest-circus-allure-environment](https://github.com/ryparker/jest-circus-allure-environment) under MIT license.
+
+<!-- TOC -->
+
+- [Jest Circus Allure Reporter](#jest-circus-allure-reporter)
+    - [❗️ Requirements](#%EF%B8%8F-requirements)
+    - [:rocket: How to use](#rocket-how-to-use)
+    - [:camera_flash: Allure reporting in your tests](#camera_flash-allure-reporting-in-your-tests)
+    - [:gear: Options](#gear-options)
+    - [📈 DocBlocks](#-docblocks)
+        - [🔍 Pragmas](#-pragmas)
+        - [🏷 Tag](#%F0%9F%8F%B7-tag)
+        - [👥 Owner](#-owner)
+        - [:part_alternation_mark: Severity](#part_alternation_mark-severity)
+        - [📇 Behaviors epics, features, stories](#-behaviors-epics-features-stories)
+
+<!-- /TOC -->
+
+
+---
+
+## ❗️ Requirements
+
+| Resource                                                             | Description                                                                                  |
+| -------------------------------------------------------------------- | -------------------------------------------------------------------------------------------- |
+| [Jest](https://jestjs.io/)                                           | A delightful JavaScript testing framework.                                                   |
+| [Allure 2 CLI](https://github.com/allure-framework/allure2#download) | "A Java jar command line tool that turns Allure result files into beautiful Allure reports." |
+
+## :rocket: How to use
+
+1. **Run a Calypso end-to-end test**
+
+```shell
+yarn jest specs/onboarding/signup__free.ts
```

Comment:
This spec doesn't exist anymore. This should probably be 'yarn jest specs/onboarding/ftme__write.ts'